modules/contact/src/userobjects/WeightedGapUserObject.C

```diff
@@ -107,6 +115,9 @@ WeightedGapUserObject::computeQpProperties()
 
   // To do normalization of constraint coefficient (c_n)
   _qp_factor = _JxW_msm[_qp] * _coord[_qp];
+
+  // Call child objects  QpProperties
+  computeQpPropertiesLocal();
```

Comment:
Use different pattern, where the child calls the parent class(es)'(s) compute method first